cmd/appKey.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"encoding/json"
+
+	kld "github.com//kaleido-io/kaleido-sdk-go/kaleido"
+	"github.com/spf13/cobra"
+)
+
+var appKeyCreateCmd = &cobra.Command{
+	Use:   "appKey",
```

Comment:
I can change it, but it will not be consistent with the methods and structs defined in kaleido/appkey.go
Do you want me to change those as well?

Reply:
hi Raul, it's a valid observation, sorry for not explaining it earlier, but the appkey.go code itself is pending a refactoring to change to appcreds. since that's all internal we haven't done that yet. but the CLI commands are public facing so they need to be consistent with the REST APIs. so please go ahead and make this change as well. thanks again for contributing!